js/jquery.mapael.js

```diff
@@ -1736,6 +1743,29 @@
                 }, interval
             );
         },
+        
+        /*
+          * Check for Raphael bug regarding drawing while beeing hidden (under display:none)
+          * See https://github.com/neveldo/jQuery-Mapael/issues/135
+          * @return true/false
+          *
+          * Wants to override this behavior? Use prototype overriding:
+          *     $.mapael.prototype.isPaperHidden = function() {return false;};
+          */
+        isPaperHidden: function(){
```

Comment:
That is actually a good question. After some though, I prefer to check for the actual Raphael bug than what cause the effect. This is in case (hypothetically) this bug is corrected hence the check would naturally pass. 

Performance wise, since it is done only once at creation, I think it doesn't matter.

What do you think ?